pkg/api/latest/dynakube/dynakube_props.go

```diff
@@ -125,3 +125,19 @@ func (dk *DynaKube) APIRequestThreshold() time.Duration {
 func (dk *DynaKube) IsTokenScopeVerificationAllowed(timeProvider *timeprovider.Provider) bool {
 	return timeProvider.IsOutdated(&dk.Status.DynatraceAPI.LastTokenScopeRequest, dk.APIRequestThreshold())
 }
+
+func (dk *DynaKube) IsCodeModulesStatusReady() bool {
+	if dk.OneAgent().GetCustomCodeModulesImage() != "" || dk.FF().IsPublicRegistry() {
+		if dk.OneAgent().GetCodeModulesImage() == "" {
+			log.Info("dynakube's codemodules status is not yet ready, codemodules 'imageID' status is missing", "dynakube", dk.Name)
```

Comment:
this is logged here and also eventually by the caller but with different wording. is this differentiation here really important? if not then the log can be remoed and if yes this info can be pushed up the stack in the form of an error